src/coreclr/src/vm/pgo.cpp

```diff
@@ -342,6 +479,166 @@ HRESULT PgoManager::getMethodBlockCounts(MethodDesc* pMD, unsigned ilSize, UINT3
     return E_NOTIMPL;
 }
 
+CORINFO_CLASS_HANDLE PgoManager::getLikelyClass(MethodDesc* pMD, unsigned ilSize, unsigned ilOffset, UINT32* pLikelihood, UINT32* pNumberOfClasses)
+{
+    *pLikelihood = 0;
```

Comment:
pLikelyhood couls also be a 32-bit float.
As it currently stands is is an integer percentage, where 100 means 100% and 33 means 33%

Reply:
I don't think we need that sort of precision; our histograms are approximate.